<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JarvisUi</class>
 <widget class="QDialog" name="JarvisUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="bg_1">
   <property name="geometry">
    <rect>
     <x>-120</x>
     <y>-20</y>
     <width>2051</width>
     <height>1131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/jarvis zip/B.G/Black_Template.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="gif3">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>391</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/jarvis zip/ExtraGui/initial.gif</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="gif2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>431</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/jarvis zip/ExtraGui/Code_Template.gif</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="gif4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>660</y>
     <width>451</width>
     <height>361</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/jarvis zip/VoiceReg/Aqua.gif</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="gif5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>290</y>
     <width>991</width>
     <height>601</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/LIVE WALLPAPER _ AI JARVIS - IRON-MAN _ STATUS REALM _ PC_DESKTOP _ (Download link in description).gif</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="gif6">
   <property name="geometry">
    <rect>
     <x>1490</x>
     <y>330</y>
     <width>411</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/jarvis zip/ExtraGui/Hero_Template.gif</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="gif8">
   <property name="geometry">
    <rect>
     <x>1480</x>
     <y>610</y>
     <width>421</width>
     <height>241</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/jarvis zip/ExtraGui/Health_Template.gif</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="gif1">
   <property name="geometry">
    <rect>
     <x>1500</x>
     <y>40</y>
     <width>361</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/jarvis zip/ExtraGui/Jarvis_Gui (1).gif</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="gif7">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>0</y>
     <width>311</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/jarvis zip/ExtraGui/Earth.gif</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="gif11">
   <property name="geometry">
    <rect>
     <x>1480</x>
     <y>970</y>
     <width>420</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/jarvis zip/B.G/gggf.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="gif11_2">
   <property name="geometry">
    <rect>
     <x>1480</x>
     <y>880</y>
     <width>421</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/jarvis zip/B.G/gggf.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>930</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
font: 20pt &quot;Times New Roman&quot;;
color: rgb(0, 243, 255);
border-radius:none;</string>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>910</y>
     <width>441</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 251, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>930</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
font: 20pt &quot;Times New Roman&quot;;
color: rgb(0, 243, 255);
border-radius:none;</string>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
  <zorder>bg_1</zorder>
  <zorder>label</zorder>
  <zorder>pushButton_start</zorder>
  <zorder>pushButton_exit</zorder>
  <zorder>gif3</zorder>
  <zorder>gif2</zorder>
  <zorder>gif4</zorder>
  <zorder>gif5</zorder>
  <zorder>gif7</zorder>
  <zorder>gif1</zorder>
  <zorder>gif6</zorder>
  <zorder>gif8</zorder>
  <zorder>gif11_2</zorder>
  <zorder>gif11</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
